<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelViewDockWidget</class>
 <widget class="QDockWidget" name="ModelViewDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>275</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>359</width>
    <height>105</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 9pt &quot;微软雅黑&quot; ;
color:rgb(255, 255, 255);
background-color: rgb(88, 88, 88);
</string>
  </property>
  <property name="features">
   <set>QDockWidget::AllDockWidgetFeatures</set>
  </property>
  <property name="allowedAreas">
   <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>模型观测器</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(172, 172, 172);</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="ModelManagerView" name="treeView">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="GComponent::ModelView" name="openGLWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>280</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GComponent::ModelView</class>
   <extends>QOpenGLWidget</extends>
   <header>UI/OpenGLWidget/modelview.h</header>
  </customwidget>
  <customwidget>
   <class>ModelManagerView</class>
   <extends>QTreeView</extends>
   <header location="global">UI/Item/modelmanagerview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
